src/dom/nativescript-vue-next/runtime/registry.ts

```diff
@@ -197,6 +199,10 @@ export function registerNativeElements() {
         'spinBox',
         () => new RNSpinBox(),
     )
+    registerElement<RNSvg, SvgProps>(
+        'qtsvg',
```

Comment:
I've checked this myself and it's fine as just "svg" after all – no conflicts with the Svelte language tooling.
```suggestion
        'svg',
```